ui/app/templates/components/generate-credentials.hbs

```diff
@@ -75,12 +75,6 @@
             @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}}
             @value={{date-format (get this.model attr.name) "MMM dd, yyyy hh:mm:ss a" isFormatted=true}}
           />
-        {{else if (and (get this.model attr.name) (eq attr.name "revocationTime"))}}
```

Comment:
`revocationTime` won't exist for a newly generated credential. (I had originally added this before understanding exactly when this template renders)